a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -359,3 +360,223 @@ def revert(self, vthunder, *args, **kwargs):
         except Exception as e:
             LOG.exception("Failed to revert partition create : %s", str(e))
             raise
+
+
+class TagEthernetBaseTask(task.Task):
```

Comment:
This code belongs to a10_network_task. We already have loaded network driver and all with VRID_FLOATING_IP changes, we will also have axapi decorator
All changes I see are related to networks.